### DISCUSSION TITLE
access for guest

### DISCUSSION THREAD

**Discussion by sadden3194** [netbox-community/netbox#12329] [Help Wanted!] [answered]
Hi all

is it possible to set read-only access for guest (no login needed)?
It should be possible the everyone can show the content of netbox without a login.
Thanks!

**Comment by candlerb** [comment, marked as answer]
Yes.

Firstly, you need `LOGIN_REQUIRED = False` (which should be the default anyway).  See [LOGIN_REQUIRED](https://docs.netbox.dev/en/stable/configuration/security/#login_required).

Secondly, you need `EXEMPT_VIEW_PERMISSIONS = ['*']`. See [EXEMPT_VIEW_PERMISSIONS](https://docs.netbox.dev/en/stable/configuration/security/#exempt_view_permissions).

**Reply by sadden3194** [reply]
Thanks, but a question: where I can set this options? :/

**Reply by candlerb** [reply]
`configuration.py`, usually at `/opt/netbox/netbox/netbox/configuration.py`.
See https://docs.netbox.dev/en/stable/configuration/

(Some settings can also be set in the web UI, via Admin > Config revisions, but the settings referred to above cannot)

**Reply by sadden3194** [reply]
it works! thank you!!
